filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+
+```sh
+# list available files
+curl $ADDRESS_OF_DCU/filewriter/api/1.8.0/status/files
+
+# file access
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_master.h5
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_data_<file_number>.h5
+```
+
+For further details on available commands and status parameters please refer to the [SIMPLON API documentation](https://media.dectris.com/210607-DECTRIS-SIMPLON-API-Manual_EIGER2-chip-based_detectros.pdf).
+
+## Multi-channel Data
+
+One of the main new features of FileWriter V2 is the support of multi-channel data, i.e. each image consists of multiple channels, one per enabled threshold/difference mode. Therefore, the dataset `/entry/data/data` is four-dimensional, i.e. `[number_of_images, number_of_channels, width, height]`. The channels are identified via strings given in the [_AXISNAME_ field](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) `channel` of the _NXdata_ class[^2]. For every channel name listed in this field, there exists a subgroup of the [_NXdetector_]() group of type _NXdetector_channel_[^1] with the same name followed by the suffix `_channel`. This group contains channel-specific metadata, for example `threshold_energy`, `flatfield` or `pixel_mask`.
+
+An _NXdetector_channel_ group could contain every field that is allowed for NXdetector and that makes sense to be given per channel.
+
+[^2]: Currently, _AXISNAME_ is defined to be of type [_NX_NUMBER_](https://manual.nexusformat.org/nxdl-types.html#nx-number), but there is an [open PR](https://github.com/nexusformat/NIAC/issues/97) to allow [_NX_CHAR_](https://manual.nexusformat.org/nxdl-types.html#nx-char) as well.
+
+#### Example
+
+For example, if `threshold_1`, `threshold_2` and `difference_mode` are enabled, we would get the following:
+
+```
+data: NXdata
+  @signal = "data"
+  @axes = ["image_number", "channel", ".", "."]
+  @image_number_indices = 0
+  @channel_indices = 1
+  image_number = [1, ..., number_of_images]
+  channel = ["threshold_1", "threshold_2", "difference"]
+  data = uint[height, width]
+
+detector: NXdetector
+  ...
+  threshold_1_channel: NXdetector_channel
+    threshold_energy = float
+    flatfield = float[height, width]
+    pixel_mask = uint[height, width]
+  threshold_2_channel: NXdetector_channel
+    threshold_energy = float
+    flatfield = float[height, width]
+    pixel_mask = uint[height, width]
+  difference_channel: NXdetector_channel
+    threshold_energy = float[2]
+```
+
+## List of fields
+
+A file generated within V2 format may contain the fields and attributes listed below. For further details and descriptions please refer to the provided links to the NeXus definitions.
+
+### / (root)
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| [@default](https://manual.nexusformat.org/classes/base_classes/NXroot.html?highlight=nxroot#nxroot-default-attribute) | NX_CHAR | Declares which NXentry group contains the data to be shown by default. |
+
+### /entry
```

Comment:
Link to https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-group